util/fipstools/delocate/delocate.go

```diff
@@ -566,10 +566,17 @@ func (d *delocation) processAarch64Instruction(statement, instruction *node32) (
 				symbol, offset, _, didChange, symbolIsLocal, _ := d.parseMemRef(arg.up)
 				changed = didChange
 
-				if _, knownSymbol := d.symbols[symbol]; knownSymbol {
+				if isFipsScopeMarkers(symbol) {
```

Comment:
Why did you choose to make it its own case if the code is the same as 2 branches below?

Reply:
because the symbol is a local symbol, so it would match
```
_, knownSymbol := d.symbols[symbol]
```
I do realise one can probably re-factor this code further. But I couldn't see an easy way to get something more general without messing up some other logic.